.github/workflows/clang-tidy.yml

```diff
@@ -0,0 +1,32 @@
+name: clang-tidy-check
+on:
+  pull_request:
+    paths-ignore:
+      - '.editorconfig'
+      - '.gitattributes'
+      - '.github/*_TEMPLATE/**'
+      - '.github/workflows/localisation.yml'
+      - '.gitignore'
+      - '.vscode/**'
+
+jobs:
+  clang-tidy-check:
+    runs-on: ubuntu-latest
+    container: openrct2/openrct2-build:12-jammy
+    steps:
+    - uses: actions/checkout@v4
+    - name: ccache
+      uses: hendrikmuhs/ccache-action@v1
+      with:
+          key: linux-clang
+    - name: Setup CCache environment
+      run: |
+        export PATH="/usr/lib/ccache:/usr/local/opt/ccache/libexec:$PATH"
+    - name: Get clang-tidy
+      run: |
+        apt-get update
+        apt-get install -y clang-tidy
```

Comment:
I will update our base image to include that, probably later this week

Reply:
Great, thanks.